src/lightning_utilities/core/imports.py

```diff
@@ -95,7 +95,9 @@ def _check_requirement(self) -> None:
                 self.message = f"Requirement {self.requirement!r} met"
             except Exception as ex:
                 self.available = False
-                self.message = f"Requirement {self.requirement!r} not met, {ex.__class__.__name__}: {ex}"
+                self.message = (
+                    f"{ex.__class__.__name__}: {ex}. HINT: Try running `pip install {self.requirement!r}`
```

Comment:
I wonder if we could add a source that redirects to installation instructions for each package. It might be overkill...

Reply:
How would we find those?

Reply:
I think overkill. One could add an optional argument to RequirementCache with this information, but then I think the responsibility grows a bit beyond just being a "requirement cache".